_episodes/02-sql-aggregation.md

```diff
@@ -118,16 +118,18 @@ about species with a count higher than 10:
 The `HAVING` keyword works exactly like the `WHERE` keyword, but uses
 aggregate functions instead of database fields to filter.
 
-If you use `AS` in your query to rename a column, `HAVING` you can use this
-information to make the query more readable. For example, in the above
-query, we can call the `COUNT(species_id)` by another name, like
-`occurrences`. This can be written this way:
+The `AS` keyword can be used to assign an alias to a column or a table.
+For example, in the above query, we can call the `COUNT(species_id)` by
+another name, like `occurrences`. This can be written this way:
```

Comment:
Note that the `AS` keyword is already introduced above, and it doesn't do anything special in the `HAVING` case (aliases work also in `JOIN`, `WHERE` or `ORDER BY` clauses). So I think I liked the previous wording better ("If you use AS...").

How about:
```
You can use the `AS` keyword to assign an alias to a column or table, and refer to that alias in the `HAVING` clause.
```

Maybe we could add more examples of using aliases in different clauses to the previous section, "Aliases"?

Reply:
Thanks for the feedback. I updated the wording as requested.